smart-contracts/osmosis/contracts/cl-vault/src/vault/range.rs

```diff
@@ -30,11 +28,6 @@ use osmosis_std::types::osmosis::{
 };
 use std::str::FromStr;
 
-/// This function is the entrypoint into the dsm routine that will go through the following steps
-/// * how much liq do we have in current range
-/// * so how much of each asset given liq would we have at current price
-/// * how much of each asset do we need to move to get to new range
-/// * deposit up to max liq we can right now, then swap remaining over and deposit again
```

Comment:
do we want to update the comment with current logic

Reply:
this is removed in the other PR